manifests/a/Azul/Zulu/15/15.38.17/Azul.Zulu.15.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Azul.Zulu.15
+PackageVersion: 15.38.17
+PackageLocale: en-US
+Publisher: Azul Systems, Inc.
+PublisherUrl: https://www.azul.com/
+PublisherSupportUrl: https://www.azul.com/support/
+PrivacyUrl: https://www.azul.com/privacy-policy/
+Author: Azul Systems, Inc.
+PackageName: Azul Zulu JDK 16
```

Comment:
```suggestion
PackageName: Azul Zulu JDK 15
```
![image](https://user-images.githubusercontent.com/11600822/150230486-0b4b41b0-07f5-47a6-93de-d49fdb7ffd95.png)

Reply:
Aaaand the damn copy pasta got past when i "correct" the packagename..